src/components/Accounts/Actions/ActionModal.js

```diff
@@ -345,16 +340,12 @@ class ActionModal extends React.Component {
     } = getState();
 
     const selected = calculateSelectedAmount(accounts, this.isRefundAction(action), feeFineActions);
-    const ownersServicePoint = [];
-    owners.forEach((item) => {
-      if (item.servicePointOwner.filter(s => s.value === curServicePoint.id).length > 0) {
-        ownersServicePoint.push(item);
-      }
-    });
-    const ownerOptions = ownersServicePoint.filter(o => o.owner !== 'Shared').map(o => ({ value: o.id, label: o.owner }));
+    const ownerOptions = owners.filter(o => o.owner !== 'Shared').map(o => ({ value: o.id, label: o.owner }));
```

Comment:
Can we move 'Shared' to const and use it from const?

Reply:
Good idea, but refactoring is out of scope of this issue.